src/components/legend/attributes.js

```diff
@@ -92,6 +92,13 @@ module.exports = {
             'or remain *constant* independent of the symbol size on the graph.'
         ].join(' ')
     },
+    itemwidth: {
+        valType: 'number',
+        dflt: 30,
+        role: 'style',
+        editType: 'legend',
+        description: 'Sets the width (in px) of the legend item symbols (the part other than the text).',
```

Comment:
Some legend icons have text on them. To avoid confusion, please replace `text` in the description with `title.text`.
Thank you!